solutions/platinum/ioi-14-wall.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: ioi-14-wall
+source: IOI
+title: 2014 - Wall
+author: Justin Ji
+---
+
+<Spoiler title = "Hint">
```

Comment:
oh there's actually an official editorial for this problem iirc

Reply:
https://ioi.te.lv/locations/ioi14/contest/day1/wall/wall-solution.pdf

Reply:
HOW DO I KEEP ON FORGETTING THIS WTF

Reply:
i’ll add it later lol

Reply:
done, i can't believe how i keep on forgetting this